src/tiled/editableworld.cpp

```diff
@@ -41,6 +41,21 @@ bool EditableWorld::containsMap(const QString &fileName) const
     return world()->containsMap(fileName);
 }
 
+bool EditableWorld::containsMap(EditableMap *map) const
+{
+    if (!map) {
+        ScriptManager::instance().throwNullArgError(0);
+        return false;
+    }
+
+    if (map->fileName().isEmpty()) {
+        ScriptManager::instance().throwError(QCoreApplication::translate("Script Errors", "Save this TileMap before checking for its presence in worlds"));
```

Comment:
Actually, if the map is not saved, couldn't we just return false without throwing an error?

Reply:
Done